src/zenml/orchestrators/utils.py

```diff
@@ -178,8 +160,24 @@ def get_config_environment_vars(
     return environment_vars
 
 
-def populate_run_name_template(run_name_template: str) -> str:
+def get_run_name(run_name_template: str) -> str:
+    """Fill out the run name template to get a complete run name.
+
+    Args:
+        run_name_template: The run name template to fill out.
+
+    Raises:
+        ValueError: If the run name is empty.
+
+    Returns:
+        The run name derived from the template.
+    """
     date = datetime.utcnow().strftime("%Y_%m_%d")
     time = datetime.utcnow().strftime("%H_%M_%S_%f")
 
-    return run_name_template.format(date=date, time=time)
+    run_name = run_name_template.format(date=date, time=time)
+
+    if run_name == "":
```

Comment:
When would this ever be reached? Only if the user specifies an empty string as run name manually? In that case, I think that check should happen even earlier, no?

Reply:
Never :P
As you said, we already catch this earlier during compilation. I just added this check here in the function in case we ever decide to use it somewhere else.